test/e2e/managedcluster_test.go

```diff
@@ -25,5 +31,117 @@ var _ = ginkgo.Describe("Testing ManagedCluster", func() {
 				return nil
 			}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 		})
+
+		ginkgo.It("Get CA from apiserver", func() {
+			//Create a fake secret for apiserver
+			fakesecretName := "fake-server-secret"
+			fakeSecret, err := e2eutil.CreateFakeTlsSecret(kubeClient, fakesecretName, utils.OpenshiftConfigNamespace)
```

Comment:
why do we need a fake secret?

Reply:
I need to fake secret as serving certs, and add it to apiserver to test get ca from apiserver.

Reply:
should cleanup the fake resource after the case.

Reply:
sure